### DISCUSSION TITLE
Possibilité d'obtenir des licences JetBrains

### DISCUSSION THREAD

**Discussion by Lenormju** [meetup-python-grenoble/meetup-python-grenoble.github.io#101] [Ideas]
Pour les User Groups il est possible d'avoir des licences JetBrains (PyCharm) pour les orgas et des participants, à creuser sur https://www.jetbrains.com/community/user-groups/#section=communities

**Comment by rclement** [comment]
Merci @Lenormju pour l'info, on va creuser l'idée !
